valhalla/userrequests/templates/userrequests/userrequest_list.html

```diff
@@ -147,4 +151,9 @@
       $('.filter-block').toggle(300);
     })
   </script>
+  <script type="text/javascript">
```

Comment:
No need for a new `<script></script>` block here. Just add the code to the one above.